Cargo.toml

```diff
@@ -24,13 +24,14 @@ async_std = ["async-std"]
 mysql = ["sqlx/mysql", "sqlx/json"]
 
 [dependencies]
-async-session = "3.0.0"
-sqlx = { version = "0.6.2", features = ["chrono"] }
+async-session = { git = "https://github.com/http-rs/async-session", branch = "main"}
+sqlx = { version = "0.7.1", features = [ "time" ] }
```

Comment:
What's the logic for switching to `time` here? That seems unrelated to upgrading.

Reply:
Due to `async-session` moving away from `chrono` and to `time`
https://github.com/http-rs/async-session/pull/35